nemo/collections/asr/parts/utils/transcribe_utils.py

```diff
@@ -289,9 +290,26 @@ def write_transcription(
     else:
         pred_text_attr_name = 'pred_text'
 
+    if isinstance(transcriptions[0], rnnt_utils.Hypothesis):  # List[rnnt_utils.Hypothesis]
+        best_hyps = transcriptions
+        assert cfg.ctc_decoding.beam.return_best_hypothesis, "Works only with return_best_hypothesis=true"
```

Comment:
Just realized this should be rnnt instead ctc here. Can we fix it, please? @karpnv

Reply:
Actually I'm not sure about hybrid ctc rnnt model when we call `write_transcription` inside `transcribe_speech.py` for that model. Any thought? @VahidooX  @titu1994

Reply:
actually `write_transcription` should handle both ctc and rnnt decoding.

Reply:
This part works for both ctc_decoding and rnnt_decoding. Just remove assert will fix it